django-server/server/admin/admin.py

```diff
@@ -0,0 +1,26 @@
+from django.apps import apps
+from django.conf.urls import url
+from django.contrib.admin import AdminSite
+
+from .views import StatsView
+
+from api.models import *
+
+class MyAdminSite(AdminSite):
+    index_template = 'admin/index_override.html'
+
+    def get_urls(self):
+        urls = super(MyAdminSite, self).get_urls()
+        custom_urls = [
+            url('stats/$', self.admin_view(StatsView.as_view(admin_site=self)), name='stats'),
+        ]
+        return urls + custom_urls
+
+
+admin_site = MyAdminSite(name='admin')
+admin_site.register(Cafeteria)
+admin_site.register(CameraEvent)
+admin_site.register(Camera)
+admin_site.register(FixedMenuOption)
+admin_site.register(FixedMenuOptionReview)
+admin_site.register(MenuOptionTag)
```

Comment:
Is it possible to include / import / source it from /api if it makes sense?

Reply:
We could load all models from all apps, but it really loads all the models, including things like activity log model.

I would leave it as is since we will need to pass some extra parameters for improving looks of particular model pages (e.g. to show status per camera).